packages/frames.js/src/frame-parsers/farcaster.ts

```diff
@@ -0,0 +1,134 @@
+import type { CheerioAPI } from "cheerio";
+import type { Frame } from "../types";
+import { isValidVersion } from "../utils";
+import {
+  getMetaTag,
+  parseButtons,
+  validate,
+  validateAspectRatio,
+  validateFrameImage,
+  validateInputText,
+  validateState,
+  validateUrl,
+} from "./utils";
+import type { ParseResult, ParsedFrame, Reporter } from "./types";
+
+type Options = {
+  reporter: Reporter;
+  fallbackPostUrl: string;
+};
+
+export function parseFarcasterFrame(
+  $: CheerioAPI,
+  { fallbackPostUrl, reporter }: Options
+): ParseResult {
+  const parsedFrame: ParsedFrame = {
+    version: getMetaTag($, "fc:frame"),
+    ogImage: getMetaTag($, "og:image"),
+    image: getMetaTag($, "fc:frame:image"),
+    imageAspectRatio: getMetaTag($, "fc:frame:image:aspect_ratio"),
+    inputText: getMetaTag($, "fc:frame:input:text"),
+    postUrl: getMetaTag($, "fc:frame:post_url") ?? fallbackPostUrl,
+    state: getMetaTag($, "fc:frame:state"),
+  };
+  const frame: Partial<Frame> = {};
+  const pageTitle = $("title").text();
+
+  if (!pageTitle) {
+    reporter.warn(
+      "<title>",
+      "A <title> tag is required in order for your frames to work in Warpcast"
+    );
+  }
```

Comment:
i don't think this is true. do you have a source for this?

Reply:
this was in the code originally so I just copied it here.

Reply:
I'm more than happy to remove it.

Reply:
got it. let me double check that it's not required but i am fairly sure it's not

Reply:
Ok, let me know when you have an answer and I will remove the warning.

Reply:
can confirm title is not required

Reply:
removed